app/controllers/payments_controller.rb

```diff
@@ -4,10 +4,10 @@ class PaymentsController < ApplicationController
   skip_after_action :verify_authorized
 
   def new
-    iugu = Iugu.new(token: ENV['IUGU_API_TOKEN'])
-    iugu.customer.create(
-      email: current_user.email,
-      name: "#{current_user.first_name} #{current_user.last_name}"
-    )
+    # iugu = Iugu.new(token: ENV['IUGU_API_TOKEN'])
```

Comment:
I was about to merge this PR :) whenever you don't want people to merge a PR, put `WIP` in the name ;)

Reply:
sorry! My mistake xD